modules/failing-to-load-the-kernel-driver.adoc

```diff
@@ -11,28 +11,29 @@ Consider the following Collector log:
 
 [source,terminal]
 ----
-[INFO    2022/10/13 14:25:13] Hostname: 'hostname'
+[INFO    2023/05/13 14:25:13] Hostname: 'hostname'
 [...]
-[INFO    2022/10/13 14:25:13] Successfully downloaded and decompressed /module/collector.ko
-[INFO    2022/10/13 14:25:13]
-[INFO    2022/10/13 14:25:13] This product uses kernel module and ebpf subcomponents licensed under the GNU
-[INFO    2022/10/13 14:25:13] GENERAL PURPOSE LICENSE Version 2 outlined in the /kernel-modules/LICENSE file.
-[INFO    2022/10/13 14:25:13] Source code for the kernel module and ebpf subcomponents is available upon
-[INFO    2022/10/13 14:25:13] request by contacting support@stackrox.com.
-[INFO    2022/10/13 14:25:13]
+[INFO    2023/05/13 14:25:13] Successfully downloaded and decompressed /module/collector.o
+[INFO    2023/05/13 14:25:13]
+[INFO    2023/05/13 14:25:13] This product uses kernel module and ebpf subcomponents licensed under the GNU
+[INFO    2023/05/13 14:25:13] GENERAL PURPOSE LICENSE Version 2 outlined in the /kernel-modules/LICENSE file.
+[INFO    2023/05/13 14:25:13] Source code for the kernel module and ebpf subcomponents is available upon
+[INFO    2023/05/13 14:25:13] request by contacting support@stackrox.com.
```

Comment:
Sorry, I know this content wasn't changed, but I was wondering if we should still have the `support@stackrox` email in the info logs? Is there a RH support email we should use instead? Logs are probably outside the scope of this but I thought I would ask.

Reply:
In fact this has already changed in collector to link directly to the repository - I'll make the necessary changes here